assets/helpers/abtest.js

```diff
@@ -15,9 +15,11 @@ const MVT_MAX: number = 1000000;
 
 // ----- Types ----- //
 
-type Audience = {
-  offset: number,
-  size: number,
+type Audiences = {
```

Comment:
So if we want the test to run across all countries what do we do? Add a separate entry for each?

Reply:
Yes, since you can test it only for the 100% of US and only 20% of US.

Reply:
But you don't repeat the variants. The test must have the same variants but can have a different segment of audiences. [Here there is an example](https://github.com/guardian/support-frontend/pull/232/files#diff-b2eddcca64ba5cda9b9632e9fe1570ddR98).

Reply:
Yes I just noticed that.